src/blocks/StickyHighlights/Highlight/index.tsx

```diff
@@ -121,13 +124,13 @@ export const StickyHighlight: React.FC<Props> = React.memo(
                         </div>
                       </div>
                     </div>
-                  </Cell>
-                </Grid>
+                  </div>
+                </div>
               </React.Fragment>
             )}
             {type === 'media' && typeof media === 'object' && media !== null && (
-              <Grid>
-                <Cell cols={6} start={7} colsM={8} startM={1}>
+              <div className={['grid'].filter(Boolean).join(' ')}>
+                <div className={'cols-6 start-7 cols-m-8 start-m-1'}>
```

Comment:
@paulpopus Are we sure the values here remain the same even with the new `column` count of `16`?

Reply:
Fixed these two